### ISSUE TITLE
translator_id does not include version hash

### ISSUE THREAD

**Issue by yusefnapora** [mediachain/schema-translators#8]
The translators currently have hardcoded id strings with version numbers, e.g. `GettyTranslator/0.1`, which doesn't fit with the new-style hash-based versioning system.

Since the version hash can't be known ahead of time, we obviously can't include it in the translator source itself.  So it seems like the solution is to remove the `/0.1` bit and have the writer accept both the translator and the version hash.  Then it can combine the un-versioned translator id + version hash into e.g `getty@QmWsU4MrF28SrYz5Kwdd2i1d1fUozakdUUuHaSUHjNrBcT` when writing out the objects.

**Comment by parkan** [comment]
Hmm I think `get_translator` can actually populate the translator with the version, so it's encapsulated

**Comment by yusefnapora** [comment]
yeah, that's a good idea.  we can inject the version into the translator there
